pubspec.yaml

```diff
@@ -39,7 +39,6 @@ dev_dependencies:
   lints: ^2.0.0
   test: ^1.15.7
   #flutter_test: {sdk: flutter}
-
```

Comment:
That does not look like an intentional change.

Reply:
Hm, looks like vs code is doing this automatically with the formatting plugin. So I would tend to just leave it in if it's okay for you :D

Reply:
Sounds good, I approved, just wanted to point out on it.